### ISSUE TITLE
Exporting and loading_indicator not working when both included in code.

### ISSUE THREAD

**Issue by mukeshbadgujar** [amcharts/amcharts5#488]
**Bug description**
I am using Exporting(Plugin) and loading_indicator (to show loading screen before chart is plot) in my project,

When i just using Exporting,
```
// Data Exporting
var exporting = am5plugins_exporting.Exporting.new(root, {
    menu: am5plugins_exporting.ExportingMenu.new(root, {}),
    dataSource: data
});
```

above code works good as needed.

if i add loading_indicator code in project (I have tried to put different location for this block of code in file.)

```
        //*****************************
        // Create a loading indicator
        loading_indicator = root.container.children.push(am5.Container.new(root, {
            width: am5.p100,
            height: am5.p100,
            layer: 1000,
            background: am5.Rectangle.new(root, {
            fill: am5.color(0xffffff),
            fillOpacity: 0.7
            })
        }));

        loading_indicator.children.push(am5.Label.new(root, {
            text: "Loading...",
            fontSize: 25,
            x: am5.p50,
            y: am5.p50,
            centerX: am5.p50,
            centerY: am5.p50
        }));

        var hourglass = loading_indicator.children.push(am5.Graphics.new(root, {
            width: 32,
            height: 32,
            fill: am5.color(0x000000),
            x: am5.p50,
            y: am5.p50,
            centerX: am5.p50,
            centerY: am5.p50,
            dy: -45,
            svgPath: "M12 5v10l9 9-9 9v10h24V33l-9-9 9-9V5H12zm20 29v5H16v-5l8-8 8 8zm-8-12-8-8V9h16v5l-8 8z"
        }));

        hourglassanimation = hourglass.animate({
            key: "rotation",
            to: 180,
            loops: Infinity,
            duration: 2000,
            easing: am5.ease.inOut(am5.ease.cubic)
        }); 
        //************************
```

Then, Just loading_indicator is working and Exporting is not working,

if i click on any option in Exporting then it gets page in infinite loop and after some time page gets crash.

please let me know if it is not bug, or its just my mistake anywhere in the code.

**Environment **

amCharts version: am5 Latest version (4.10.26)
Browser and its version : Version 102.0.5005.61 (Official Build) (64-bit)
Related frameworks and their versions: vanilla JavaScript

**Comment by mukeshbadgujar** [comment]
> Normally, Exporting waits for everything to finish rendering in chart to proceed.
> 
> Since you have a perpetual hourglass animation running, chart is in perpetual state of change, hence Exporting waiting forever.
> 
> I imagine you are not exporting loading indicator, it's hidden when export happens. So why not pause the animation when you hide the indicator?
> 
> ```JavaScript
> hourglassanimation.pause();
> ```

When the chart is in loading state for getting data from back-end that time i start and pause the hourglassanimation  and hide the loading indicator.

So when chart is loaded the hourglassanimation is already in pause state, so i don't think that it again need to run "hourglassanimation.pause()" before the Exporting operation is trigger.

**Comment by martynasma** [comment]
Do you mean export does not work when indicator animation is already paused?

If that is so, would you be able to provide a working repro case in CodePen or jsFiddle?

**Comment by mukeshbadgujar** [comment]
I have written necessary code, but jsfiddle and CodePen getting not responding after running. (i dont know why??) 

check the code click on start stop buttons and load button to load chart.
https://jsfiddle.net/hd8kman0/2/

**Comment by mukeshbadgujar** [comment]
> export

Yes, Exactly i am saying.

please check above code on jsfiddle

**Comment by martynasma** [comment]
Well, it seems you create `hourglassanimation` twice.

So you pause one, but the other one keeps on playing.

**Comment by mukeshbadgujar** [comment]
its my typing mistake in previous code,

please check this code,

https://jsfiddle.net/ekbnqg06/

process to generate issue:

1. click on load chart button (it will load chart and disable the loader)
2. you can also start and stop loading indicator with start and stop button 
3. after stopping the loading indicator, click on exporting icon and click on any option
4. your site will be freeze after clicking on option.

**Comment by mukeshbadgujar** [comment]
> Well, it seems you create `hourglassanimation` twice.
> 
> So you pause one, but the other one keeps on playing.

Please check my above comment.

**Comment by martynasma** [comment]
Indeed. Pausing an animation still makes it go into a dead loop.

We're looking into it.

In the interim, you can work around it by using `stop()` instead of `pause()`.

The downside is that if you will need to show indicator again, you will need to create a new animation.

**Comment by mukeshbadgujar** [comment]
Ok I understand,

I will wait for new update,

Till then i will make it stop().

Let me know when it will be fixed...

Thanks....

**Comment by martynasma** [comment]
Fixed in [5.2.14](https://github.com/amcharts/amcharts5/releases/tag/5.2.14).

## [5.2.14] - 2022-07-23

### Added
- New setting `keepTargetHover` added to `Tooltip` (default: `false`). If set to `true`, hovering the tooltip will be treated like hovering its target element itself.

### Fixed
- Removing `verticalScrollbar` from a `Container` was causing some issues.
- Labels with HTML content and horizontal padding were not being sized/centered properly.
- Date formatting code "EEEE" was producing weekday's short version ("Tue") instead of full one ("Tuesday").
- `Scrollbar` was not working after it was zoomed and clicked on a background to change position, if no animated theme was used.
- Paused animations will no longer obstruct chart export.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
